platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/TestEnvironment.java

```diff
@@ -52,4 +52,8 @@ public interface TestEnvironment {
      * process to complete causing the test to fail.
      */
     void destroy() throws InterruptedException;
+
+    void start();
+
+    void stop();
```

Comment:
These methods have to be removed. The user should not be responsible for starting and stopping the environment.